### ISSUE TITLE
Fid score 

### ISSUE THREAD

**Issue by qsc227** [lyndonzheng/TFill#18]
Hello，Thanks for your code! 
But I have some questions ,
why i use the given pre-trained models to test on the test sets of CelebHQ, and i calculate the fid score is 5.03.
i use your code (def center_mask()) to generate the test sets with center masks .
could you please tell me what is the problem?
